mkosi/__init__.py

```diff
@@ -6601,11 +6619,12 @@ def find_address(args: CommandLineArguments) -> Tuple[str, str]:
 
 
 def run_ssh(args: CommandLineArguments) -> None:
-    assert args.output_sshkey is not None
+    ssh_key = args.ssh_private_key if args.ssh_private_key else args.output_sshkey
```

Comment:
```suggestion
    ssh_key = args.ssh_private_key or args.output_sshkey
```
This is also compatible with the following assert, since if both or `None`, the return will be `None`.

Reply:
Perfect, thanks, that looks a way better.